src/Trash.svg

```diff
@@ -1,3 +1,6 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M21 4V6H14H11H3V4H9L10 3H14L15 4H21ZM17 19V7H19V19C19 20.1 18.1 21 17 21H7C5.9 21 5 20.1 5 19V7H7V19H17ZM9 17H11V7H9V17ZM15 17H13V7H15V17Z" fill="black"/>
+<path d="M21 4V6H3V4H9L10 3H14L15 4H21Z" fill="black"/>
+<path d="M17 19V7H19V19C19 20.1 18.1 21 17 21H7C5.9 21 5 20.1 5 19V7H7V19H17Z" fill="black"/>
+<path d="M9 17H11V7H9V17Z" fill="black"/>
+<path d="M15 17H13V7H15V17Z" fill="black"/>
 </svg>
```

Comment:
The updated image looks as if the shapes have gotten off-pixel as compared to the previous version, which looks sharper.

![CleanShot 2020-10-14 at 19 03 45](https://user-images.githubusercontent.com/545605/96058063-ffb21c00-0e4f-11eb-9ecc-8b53da500498.png)

Reply:
It doesn't look like the points have changed (not sure why the other `d` data did) and I'm not seeing a difference in the swipe preview

Reply:
It might just be anitaliasing and pixel rounding on my screen. Please disregard.